dowhy/causal_estimators/regression_estimator.py

```diff
@@ -68,11 +66,11 @@ def _build_features(self, treatment_values=None, data_df=None):
         # Using all data by default
         if data_df is None:
             data_df = self._data
-            treatment_vals = self._treatment
+            treatment_vals = pd.get_dummies(self._treatment, drop_first=True)
```

Comment:
This is a good solution, but I think it will be useful if the user can specify which value of treatment will be assigned to control. Will this work if we instruct users to always keep the control as the first treatment column? Not sure how `drop_first` is interpreted.

Reply:
The `get_dummies` function will just drop the first variable encountered, so yes we should tell the user to make the first row of the dataset correspond to control. If that is a bit messy, we could instead switch over to `sklearn`s OneHotEncoder, which I think is more flexible

Reply:
Got it. I think this is okay as a first implementation, especially since `get_dummies` is used in many other places.